tests/Resources/ODS.robot

```diff
@@ -68,3 +71,36 @@ Usage Data Collection Should Not Be Enabled
     [Arguments]    ${msg}="Usage Data Collection" should not be enabled
     ${enabled}=    ODS.Is Usage Data Collection Enabled
     Should Not Be True    ${enabled}    msg=${msg}
+
+Uninstall RHODS From OSD Cluster
+    [Documentation]    Selects the cluster type and triggers the RHODS uninstallation
+    ${addon_installed}=     Is Rhods Addon Installed    ${CLUSTER_NAME}
+    Uninstall RHODS Using OLM
+    # IF    ${addon_installed} == ${TRUE}
+    #     Uninstall Rhods Using Addon    ${CLUSTER_NAME}
+    # ELSE
+    #     Uninstall RHODS Using OLM
+    # END
+
+Uninstall RHODS Using OLM
+    Selected Cluster Type OSD
+    Uninstall RHODS
+
+Wait Until RHODS Uninstallation Is Completed
```

Comment:
Documentation missing.  You can consider to add the _Private_ tag  and call it from `Uninstall RHODS From OSD Cluster`

Reply:
In the ODS-CI Style guide:  _Private: indicates that the keyword cannot be used directly by test cases/test suites, as it’s meant to be used internally by the library defining it._

Reply:
yes yes but I mean why should it be private?

Reply:
If you were calling that keyword from `Uninstall RHODS From OSD Cluster `  the  tests wouldn't need it, so it could be private. Not a  big deal ...